course/lesson-2-your-first-error/lesson.tres

```diff
@@ -110,6 +112,11 @@ goal = "This code is incorrect and will cause an error when you try to run it.
 
 The code defines an empty function named [code]this_code_is_wrong[/code].
 
+The code also includes a comment. Comments look like this:
+[code]#this is a comment[/code]
+
+Comments are a way to add notes for ourselves or other coders, when they read the code nearby. They aren't \"run\" by the computer and there is no limit to how many comments we can add to a program.
+
```

Comment:
Not sure about this, first I'd use the exact same comment line `#return`, then I find the existing practice pitch heavy already, so I'd look for a way to rephrase and shorten the whole thing rather than add extra. We haven't had anyone report they got stuck or had trouble here, including our beginners testers, so I'm not even sure if anyone is confused by the existing explanation.